src/core/lib/iomgr/ev_posix.c

```diff
@@ -163,11 +166,6 @@ void grpc_fd_notify_on_write(grpc_exec_ctx *exec_ctx, grpc_fd *fd,
   g_event_engine->fd_notify_on_write(exec_ctx, fd, closure);
 }
 
-grpc_pollset *grpc_fd_get_read_notifier_pollset(grpc_exec_ctx *exec_ctx,
```

Comment:
Let's wait on removing this: I want to do some timings with it once the SO_REUSEPORT stuff gets in.